core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.constants.ts

```diff
@@ -23,72 +23,44 @@ export class TopicLandingPageConstants {
   public static TOPIC_LANDING_PAGE_DATA = {
     math: {
       fractions: {
-        topic_title: 'Fractions',
-        topic_tagline: 'Add, Subtract, Multiply and Divide',
-        collection_id: '4UgTQUc1tala',
-        page_data: {
-          image_1: {
-            file_name: 'matthew_paper.png',
-            alt: 'Matthew showing parts of fractions written on a card.'
-          },
-          image_2: {
-            file_name: 'matthew_fractions.png',
-            alt: 'Matthew solving problems on Oppia.'
-          },
-          video: 'fractions_video.mp4',
-          lessons: [
-            'What is a Fraction?',
-            'Comparing Fractions',
-            'The Meaning of "Equal Parts"',
-            'Adding & Subtracting Fractions'
-          ]
-        }
+        topicTitle: 'Fractions',
+        topicTagline: 'Add, Subtract, Multiply and Divide',
+        collectionId: '4UgTQUc1tala'
       },
       'negative-numbers': {
-        topic_title: 'Negative Numbers',
-        topic_tagline: '',
-        collection_id: 'GdYIgsfRZwG7',
-        page_data: {
-          image_1: {
-            file_name: 'negative_1.png',
-            alt: 'A boy showing 3 + -24 written on a slate.'
-          },
-          image_2: {
-            file_name: 'negative_2.png',
-            alt: 'A boy smiling and solving negative-number problems on Oppia.'
-          },
-          video: 'negative-numbers_video.mp4',
-          lessons: [
-            'The Number Line',
-            'What is a Negative Number?',
-            'Adding & Subtracting Negative Numbers',
-            'Multiplying & Dividing Negative Numbers'
-          ]
-        }
+        topicTitle: 'Negative Numbers',
+        topicTagline: '',
+        collectionId: 'GdYIgsfRZwG7'
       },
       ratios: {
-        topic_title: 'Ratios',
-        topic_tagline: '',
-        collection_id: '53gXGLIR044l',
-        page_data: {
-          image_1: {
-            file_name: 'ratios_James.png',
-            alt: 'A boy showing 2 is to 3 ratio on a card.'
-          },
-          image_2: {
-            file_name: 'ratios_question.png',
-            alt: 'A smoothie shop and a card having question "What does a' +
-              ' ratio tell us?" with options.'
-          },
-          video: 'ratios_video.mp4',
-          lessons: [
-            'What is a Ratio?',
-            'Equivalent Ratios',
-            'Ratios & Proportional Reasoning',
-            'Writing Ratios in Simplest Form'
-          ]
-        }
+        topicTitle: 'Ratios',
+        topicTagline: '',
+        collectionId: '53gXGLIR044l'
       }
     }
   };
+
+  public static LESSON_QUALITIES_DATA = [{
+    title: 'Fun storytelling',
+    description: (
+      'Oppia\'s lessons tell stories using video and images to ' +
+      'help learners apply math concepts in everyday life.'),
+    imageFilename: 'fun_storytelling.png',
+    imageAlt: 'Storytelling lessons presentation.'
```

Comment:
Alt text should describe the image. E.g. "An image of a lesson journey illustrated on top of an open book.", "A speaker emitting sound next to text and a Play button", "A selection of characters from the lessons."